runtime/doc/lua.txt

```diff
@@ -790,7 +790,7 @@ vim.json.decode({str}, {opts})                             *vim.json.decode()*
 
     Parameters: ~
       • {str}   (string)
-      • {opts}  table<string,|nil any>
+      • {opts}  table<string,? any>
```

Comment:
These are pre-existing errors. I plan on adjusting types in every module, but fixing them in this PR could lead to scope creep. So unless anyone objects, I will leave this as-is.